sumo_mgmt.py

```diff
@@ -743,3 +755,7 @@ def delete_offline_collectors():
             print_sources_table(collectors_with_sources, source_table_headings)
         elif args.deleteOfflineCollectors and prompt(msg):
             delete_offline_collectors()
+        elif args.assignBudget:
+            for collector in collectors:
+                log('Assigning budgetId ' + args.budgetId[0] + ' to collector ' + str(collector['name']))
+                add_Ingest_Budget(args.assignBudget[0],int(collector['id']))
```

Comment:
do we need to sleep to avoid API throttling like we do in https://github.com/SumoLogic/collector-management-client/blob/c627a85692d3d747c6caafb7987eeab0638f2cec/sumo_mgmt.py#L329